packages/ripple-binary-codec/src/enums/definitions.json

```diff
@@ -44,6 +44,7 @@
     "NegativeUNL": 78,
     "NFTokenPage": 80,
     "NFTokenOffer": 55,
+    "Amm": 121,
```

Comment:
Why is this lowercase here? (We've been using the capitalized "AMM" in all the other naming conventions)

Reply:
Great catch. Looks like there's a bug in the xrpl-codec-gen script. I created a PR to handle this case - https://github.com/RichardAH/xrpl-codec-gen/pull/6/files